app/views/desks/show.html.erb

```diff
@@ -1,7 +1,28 @@
-<%= link_to "Back to desk list", root_path %>
-<h3>Desk details</h3>
-<ul>
-  <li><%= @desk.id %></li>
-  <li><%= @desk.name %></li>
-  <li><%= @desk.occupied %></li>
-</ul>
+<nav aria-label="breadcrumb" role="navigation">
+  <ol class="breadcrumb">
+    <li class="breadcrumb-item"><a href="<%= root_path %>">Desk Listing</a></li>
+    <li class="breadcrumb-item active" aria-current="page"><%= @desk.name %></li>
+  </ol>
+</nav>
+
+<div class="col-lg">
+<div class="card">
+
+  <div class="card-body">
+
+    <% occupied_class = 'alert-success' %>
+    <% status_text = 'Not Occupied' %>
+
+    <% if @desk.occupied? %>
```

Comment:
I feel this is too much logic for the view. I'd abstract it into a method in a helper. You can find more info in: https://mixandgo.com/blog/the-beginners-guide-to-rails-helpers